sharkfuser/include/fusilli.h

```diff
@@ -13,25 +13,28 @@
 #ifndef FUSILLI_H
 #define FUSILLI_H
 
-// Attributes
+// Support
+#include "fusilli/support/asm_emitter.h"
+#include "fusilli/support/cache.h"
+#include "fusilli/support/external_tools.h"
+#include "fusilli/support/extras.h"
+#include "fusilli/support/logging.h"
+
+// Attributes / Types
 #include "fusilli/attributes/attributes.h"
 #include "fusilli/attributes/conv_attributes.h"
 #include "fusilli/attributes/tensor_attributes.h"
+#include "fusilli/attributes/types.h"
 
 // Nodes
 #include "fusilli/node/conv_node.h"
 #include "fusilli/node/node.h"
 
-// Utilities
-#include "fusilli/asm_emitter.h"
-#include "fusilli/backend.h"
-#include "fusilli/external_tools.h"
-#include "fusilli/extras.h"
-#include "fusilli/logging.h"
-#include "fusilli/types.h"
+// Backend
+#include "fusilli/backend/backend.h"
```

Comment:
Do we want everything in a subdirectory? We could wait to break things into sub-directories until conceptual grouping of things requires more than one file. With everything in a subdirectory we have a lot of `#import backend/backend.h" and `#import graph/graph.h`.

Reply:
Shortfin does it too (putting all headers in subdirs): https://sourcegraph.com/github.com/nod-ai/shark-ai/-/tree/shortfin/src/shortfin so was going off of that. Also, right now backend has just one header but I'm anticipating all the runtime C-API interfacing also goes here. Backend here is not the compiler backend, rather fusilli's backend - so anything to do with IREE compiler/runtime could go here.